content/millennium/r4/diagnostic/observation.md

```diff
@@ -119,6 +119,35 @@ Notes:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Retrieve by id
+
+List an individual Observation by its id:
+
+    GET /Observation/M-id
```

Comment:
what about charted blood pressures?

Reply:
I have already asked Fran about this, she told told that its enough if we just include measurements as BP will be implied.